stack.yaml

```diff
@@ -0,0 +1,66 @@
+# This file was automatically generated by 'stack init'
```

Comment:
hrm, what is this Stack file? (I'm not really doing much Haskell dev these days)